src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.tsx

```diff
@@ -211,6 +212,8 @@ export default class SentryApplicationRow extends React.PureComponent<Props> {
     const {app, onInstall, organization} = this.props;
     const isInstalled = !!this.isInstalled;
 
+    recordInteraction(app, 'sentry_app_viewed');
```

Comment:
@nolachen So there is another way an app can be installed - through /views/sentryAppExternalInstallation.jsx. You can get there with the URL `/sentry-apps/:sentryAppSlug/external-install/`. The main use case is for partners to add an installation link on their website that goes directly to a view where they can install a particular sentry app. I think we should count these as views as well or else the numbers may not make sense.